vendor/engines/projects/config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   resources :facilities, only: [] do
-    resources :projects, controller: "projects/projects", except: [:destroy]
+    resources :projects, controller: "projects/projects", except: [:destroy] do
+      collection do
```

Comment:
Ah, cool. You can also write this as `get "inactive", on: :collection` as a one-liner.